### DISCUSSION TITLE
How to set min-width to a widget

### DISCUSSION THREAD

**Discussion by Abhay0294** [eclipse-theia/theia#12636] [Q&A]
Hi Team,

I have the following requirement.

- I have a right side widget which is resizable by default,

- I want it to have min-width, beyond which user cannot resize it.**[This is required to retain the widget content from getting distorted]**

Things I tried:

- Tried to handle with CSS but no luck. Here content got fixed width but still the widget is resizable, which finally distorts the view.

Requesting for suggestions to implement the same.

**Comment by msujew** [comment]
Hey @Abhay0294,

See my answer over at [the Theia discourse thread](https://community.theia-ide.org/t/how-to-set-minimum-width-of-widget-and-not-allow-resizing-below-that-width/676/6). To repeat:

Given that Theia cannot restrict the minimum browser size AND Theia never overflows, I believe it’s impossible to set a minimum size for any given widget. You would need to handle overflow in the widget yourself.

**Reply by Abhay0294** [reply]
Thanks a lot @msujew for the response.
